articles/architecture-scenarios/b2c.md

```diff
@@ -1,108 +1,144 @@
+* Custom domains
+* Tenant association
+* Support for the Software Development Life Cycle (SDLC)
+    
+### User provisioning
 
-Now when a user signs in to one of your applications, they can access other applications integrated with Auth0 without having to log in again. This will be true until their SSO session expires. You should configure the SSO session length within Auth0 to meet security policies.
+[User provisioning](/architecture-scenarios/implementation/b2c/user-provisioning) is the next workstream, and this can be done in parallel with [user authentication](#user-authentication). We've found that the most successful implementations address the following topics at this state during phase 1, however precisely what you tackle will depend on your specific requirements. For example, you may not need user migration, or you may already have an existing sign up mechanism that can be leveraged. 
 
-## Account linking
+The user provisioning workstream includes setting up:
 
-Social logins are convenient for users, but social providers may only have a few user profile attributes. You'd like to build rich user profiles, store them in Auth0, but not lose the convenience of social logins. You can do this with [account linking](/link-accounts).
+* User migration
+* User sign up
 
-Account linking lets users link one or more social logins to their Auth0 profile. This creates a merged user profile with attributes from the social provider and from the user profile from Auth0. When a user logs in through a social provider, your application sees the merged profile.
+::: panel Best Practice
+The Auth0 Dashboard in conjunction with the [Delegated Administration Extension](/extensions/delegated-admin/v3) can be used out-of-the-box to provide for user provisioning and deprovisioning as described in this [Auth0 blog post](https://auth0.com/blog/delegated-admin-v2/). If you require more comprehensive deprovisioning functionality, for compliance reasons for example, then we recommend you do so in a later phase.
+::: 
 
-## Extensibility with augmented user profiles
+### Authentication
 
-You may want to enrich user profiles with data obtained from other sources. [Auth0 Rules](/rules) enable you to write small snippets of code that execute during the authentication transaction. This lets you call other services for user information, then add it to the Auth0 user profile.
+[Authentication](/architecture-scenarios/implementation/b2c/authentication) comes next and can be done in parallel with user provisioning. Topics to address at this point include:
 
-## Passwordless login
+* Universal login
+* Username and password authentication
+* Application integration
 
```

Comment:
This list should be title case, and to echo the above comment from Peter, should probably be links.